molgenis-data-vcf/src/test/java/org/molgenis/data/vcf/utils/test/VcfUtilsTest.java

```diff
@@ -13,7 +13,6 @@
 import static org.molgenis.data.vcf.model.VcfAttributes.POS;
 import static org.molgenis.data.vcf.model.VcfAttributes.QUAL;
 import static org.molgenis.data.vcf.model.VcfAttributes.REF;
-import static org.molgenis.data.vcf.utils.VcfWriterUtils.EFFECT;
```

Comment:
Entire setup of this test is obsolete, only test that is executed (createId) requires 8 lines of setup. I will merge this PR anyway. I'll create a cleanup PR for this test.